dotCMS/src/main/java/com/dotcms/workflow/helper/WorkflowHelper.java

```diff
@@ -293,6 +293,20 @@ private BulkActionView buildBulkActionView (final Map<CountWorkflowStep, List<Wo
         return bulkActions;
     }
 
+    private Optional<WorkflowAction> findActionsOn (final String actionName, final List<WorkflowAction>  workflowActions) {
+
+        if (UtilMethods.isSet(workflowActions)) {
+
+            final Optional<WorkflowAction> foundAction  =
+                    workflowActions.stream()
+                            .filter(action -> action.getName().equalsIgnoreCase(actionName)).findFirst();
+
+            return foundAction;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Consider simply returning the value vs storing it in local variable 'foundAction'](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4801230)